### PR TITLE
Carousel height

### PR DESCRIPTION
**Pull request by curtisadams28**
ADO card: ☑️ [AB#8384](https://dev.azure.com/S72/SHIFT72/_workitems/edit/8384)

Designs: 
- 📱 [Mobile](https://zpl.io/YYdK3Rk)
- 💊 [Tablet](https://zpl.io/ykjONyy)
- 🖥 [Desktop](https://zpl.io/PqzPZYA)

- [Design Review](https://www.figma.com/file/pj55IWE8MqtBwMalHntMTg/?node-id=814:31509)

## Description of work
1. The carousel now scales up and down depending on the window height. This allows the carousel to fit properly on both small and large devices.
2. The carousel images no longer extend the full height of the carousel. This leaves a darker area under the text before the gradient transitions.

Image Height

### Affected Clients
 - All clients that upgrade to the latest version of core-template.

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [x] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/styles/_carousel.scss [hunk 1]
```diff
@@ -94,9 +96,14 @@ s72-carousel {
   width: 100%;
   z-index: 999;
 
+  @include media-breakpoint-up(sm) {
+    flex-direction: column;
```
Comment (PeculiarGoat) [inline site/styles/_carousel.scss:100]: This line is redundant right?

**Review by l0ud0gg — APPROVED** [review]
code looks fine to me. perhaps adjusting the min height for smaller phones where the url bar takes up some of the viewport height but not a big deal.

**Review by smitsanghvi21 — APPROVED** [review]

**Comment by smitsanghvi21** [comment]
Will need to resolve conflicts before merging.

File: site/styles/_carousel.scss (hunk 1 above)
Comment (curtisadams28) [reply site/styles/_carousel.scss]: Yeah. Removed.

**Review by PeculiarGoat — APPROVED** [review]
Seems alright!

**Pull request merged**